admin-dev/themes/new-theme/js/components/grid/extension/submit-grid-action-extension.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+const $ = window.$;
+
+/**
+ * Class SubmitGridActionExtension handles grid action submits
+ */
+export default class SubmitGridActionExtension {
+  constructor() {
+    return {
+      'extend': (grid) => this.extend(grid)
+    };
+  }
+
+  extend(grid) {
+    grid.getContainer().on('click', '.js-grid-action-submit-btn', (event) => {
+      this.handleSubmit(event, grid);
+    });
+  }
+
+  /**
+   * Handle grid action submit.
+   * It uses grid form to submit actions.
+   *
+   * @param {Event} event
+   * @param {Grid} grid
+   *
+   * @private
+   */
+  handleSubmit(event, grid) {
+    const $submitBtn = $(event.target);
+    const confirmMessage = $submitBtn.data('confirm-message');
+
+    if (typeof confirmMessage !== "undefined" && 0 < confirmMessage.length) {
+      if (!confirm(confirmMessage)) {
```

Comment:
You can have a single `if`

Reply:
i know, but isnt that easier to read?

Reply:
You can use multi lines that help to read =)
```
if (typeof confirmMessage !== "undefined" && 
    confirmMessage.length !== 0 && 
    !confirm(confirmMessage)
) {
    return;
}
```

Reply:
well, if you think this is better, im okay with that. :)